### PR TITLE
Add --json flag

### PR DESCRIPTION
**Pull request by kpcyrd**

### CODE REVIEW

File: src/types.rs [hunk 1]
```diff
@@ -85,8 +87,9 @@ pub struct Affected {
 }
 
 impl Affected {
-    pub fn new(package: &str) -> Self {
+    pub fn new(package: &str, avg: &Avg) -> Self {
```
Comment (anthraxx) [inline src/types.rs:90]: This isn't really correct. The way the logic works is that `Affected` is a bundle of all CVEs from all AVGs related to a certain package. Hence the CVEs list etc gets extended. Declaring one Avg for this bundle is not right, it should be a Vec of Avg's/names

Comment (kpcyrd) [reply src/types.rs]: I've updated the PR so `--json` prints the filtered AVGs instead of the `Affected` bundle.

File: src/main.rs [hunk 2]
```diff
@@ -164,7 +183,7 @@ fn _get_required_by_recursive(
 }
 
 /// Given a package and an `avg::AVG`, returns true if the system is affected
```
Comment (anthraxx) [inline src/main.rs:185]: this comment is wrong now, should mention the fixed version instead.

Comment (kpcyrd) [reply src/main.rs]: Updated!

**Comment by anthraxx** [comment]
applied via e6e6429c6b828b1aaaa303eabed6e01dce23778c
thanks a lot

**Pull request closed without merging**